pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBase.java

```diff
@@ -37,12 +37,12 @@ public abstract class CmdBase {
     private PulsarAdmin admin;
     private IUsageFormatter usageFormatter;
 
-    @Parameter(names = { "-h", "--help" }, help = true, hidden = true)
-    private boolean help;
+    @Parameter(names = { "--help", "-h" }, help = true, hidden = true)
+    private boolean help = false;
```

Comment:
Is this variable/parameter useless and can it be deleted? Because `-h` is used to print the usage of `./bin/pulsar-admin`, but it has been defined in PulsarAdminTool.RootParams
https://github.com/apache/pulsar/blob/0d707aec92595868192380758a46f8f02886128f/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminTool.java#L98-L99

Reply:
Nope. This `--help` is applied at `pulsar-admin schemas -h` level. That is, one level down.

Reply:
You may read the PR description again and try

```bash
./bin/pulsar-admin -h
./bin/pulsar-admin schemas -h
./bin/pulsar-admin schemas delete -h
```

with/without this patch.

Reply:
Thanks for your explanation. 
I will verify it base on this pr and review today.
By the way, it would be better if could add unit test :)

Reply:
> unit test

The mechanism of help flag is provided by jcommander. Thus, I think we can verify the output manually. Writing unit tests for such cases is almost testing a third-party dependency.

Reply:
But we may still test whether the help flag is set. Let me try to write one.

Reply:
Added `testHelpFlag`.